tests/xtro-sharpie/watchOS-ClockKit.ignore

```diff
@@ -0,0 +1,6 @@
+# Default constructor is disabled, not need to add it.
+!deprecated-attribute-missing! CLKFullColorImageProvider::init missing a [Deprecated] attribute
+!incorrect-protocol-member! CLKComplicationDataSource::getSupportedTimeTravelDirectionsForComplication:withHandler: is OPTIONAL and should NOT be abstract
+
+# no needed, families is a smart enum and the same can be achived with .net
+!missing-pinvoke! CLKAllComplicationFamilies is not bound
```

Comment:
I think it needs to be bound

https://docs.prod.msd.developer.g.apple.com/documentation/clockkit/3555308-clkallcomplicationfamilies

sounds like it can return a different subset based on OS version (or even watch model at some point) while our smart enum will always have the same values (and the availability attributes are removed by the linker)